packages/utils/src/snark-artifacts/snark-artifacts.browser.ts

```diff
@@ -1,31 +1,21 @@
-import { GetSnarkArtifactUrls } from "./config"
-import { Proof, SnarkArtifacts, Version } from "../types"
+import { BigNumber, SnarkArtifacts, Version } from "../types"
 
-function MaybeGetSnarkArtifacts(proof: Proof.EDDSA, version?: Version): () => Promise<SnarkArtifacts>
-function MaybeGetSnarkArtifacts(
-    proof: Proof.POSEIDON,
-    version?: Version
-): (numberOfInputs: number) => Promise<SnarkArtifacts>
-function MaybeGetSnarkArtifacts(
-    proof: Proof.SEMAPHORE,
-    version?: Version
-): (treeDepth: number) => Promise<SnarkArtifacts>
-function MaybeGetSnarkArtifacts(proof: Proof, version?: Version) {
-    switch (proof) {
-        case Proof.POSEIDON:
-            return async (numberOfInputs: number) => GetSnarkArtifactUrls({ proof, numberOfInputs, version })
+export default async function maybeGetSnarkArtifacts(
+    projectName: string,
+    options: {
+        parameters?: (BigNumber | number)[]
```

Comment:
Do you use an array because you foresee having to pass more parameters to the generate function? Right now we always use 1. So why not just
`parameters?: BigNumber | number`?

Reply:
As you are joining them below, this could turn into errors.
This allows people to pass [2, 3, 4]. For functions that don't use so many arguments

Reply:
Arrays should also handle cases where a circuit has more than 1 parameter. So, it covers all cases where N of parameters > 0, which could be encoded as `project-1-34-<...>.wasm`. We could also use `(BigNumber | number)[] | BigNumber | number` tho.